contracts/governance/src/ZeroExTreasuryGovernor.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2023 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/governance/Governor.sol";
+import "@openzeppelin/governance/extensions/GovernorSettings.sol";
+import "@openzeppelin/governance/extensions/GovernorCountingSimple.sol";
+import "@openzeppelin/governance/extensions/GovernorVotes.sol";
+import "@openzeppelin/governance/extensions/GovernorVotesQuorumFraction.sol";
+import "@openzeppelin/governance/extensions/GovernorTimelockControl.sol";
+
+import "./IZeroExVotes.sol";
+import "./ISecurityCouncil.sol";
+
+contract ZeroExTreasuryGovernor is
+    ISecurityCouncil,
+    Governor,
+    GovernorSettings,
+    GovernorCountingSimple,
+    GovernorVotes,
+    GovernorVotesQuorumFraction,
+    GovernorTimelockControl
+{
+    constructor(
+        IVotes votes,
+        TimelockController _timelock,
+        address _securityCouncil
+    )
+        Governor("ZeroExTreasuryGovernor")
+        GovernorSettings(14400 /* 2 days */, 50400 /* 7 days */, 5e11)
```

Comment:
maybe I'm missing details here, but I think the proposal threshold is meant to be `500` which would translate to `5e20` instead of the `5e11`

Reply:
The units on treasury governance voting power is in sqrt'd token balances. So the basis is 10**9, not 10**18. We'll probably be readjusting the basis back to 10**18 to support some additional features and because the difference in basis is confusing